Source/CarthageKitTests/XcodeSpec.swift

```diff
@@ -17,6 +17,8 @@ import Tentacle
 import XCDBLD
 
 class XcodeSpec: QuickSpec {
+	static let currentSwiftVersion = "3.0.2"
```

Comment:
Hey, 😄 

The approach I’m currently leaning towards is having those few tests only run on CI — have an if statement check whether the environment variable for `CI` is set and not `"false"`, then put those `it` calls from Quick inside that conditional.

That condition can also include check whether some environment variable we export where we hardcode for each Xcode we use in CI.